spec/completely/integration.yml

```diff
@@ -1,24 +1,32 @@
 ftp:
```

Comment:
Since completions are now returned differently if the input ends with a hyphen, this integration spec config was expanded to list both cases.